ccx_messaging/publishers/rule_processing_publisher.py

```diff
@@ -107,13 +106,14 @@ def publish(self, input_msg, report):
 
         try:
             msg_timestamp = input_msg["timestamp"]
+            msg_version = report.pop("version", 1)
```

Comment:
The "1" here is totally arbitrary. With the new formatter, the version (integer higher than 1) should always be included, so we could use 0 here to send something that the aggregator could complain about.

Reply:
if you do that, I would like to see a test to check that the default value returned is 0 (if possible)

Reply:
updated